src/Strat-Pairs.py

```diff
@@ -279,18 +304,18 @@ def run(self):
 
     def run_backtest(self):
         def setup_bootstrap_conditions():
-            self.pd_last_prices = pd.read_csv("ticks 10 mins - Jun 25 2014.csv")
+            self.pd_last_prices = pd.read_csv("C:\\Users\\jloss\\PyCharmProjects\\IB-Trading-Models-And-Backtester\\src\\ticks 10 mins - Jun 25 2014.csv")
```

Comment:
This line should be reverted so that it works with relative path (absolute path will work only on the authors computer)